templates/4a_compensate_image_data.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "237267d0",
```

Comment:
<p>If they like row 3, row 4, or row 5, then they're done and can pick that matrix. It's only if they want an intermediate value that they need to re-run the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>I agree here before moving on to the next step the optimize the matrix section should be here. Add is at an "Optional" step in the heading. </p>